static/js/utils/app/explore_utils.ts

```diff
@@ -40,6 +40,9 @@ export function getTopics(
   pageMetadata: SubjectPageMetadata,
   placeUrlVal: string
 ): Item[] {
+  if (_.isEmpty(pageMetadata?.childTopics)) {
```

Comment:
could there be a case where there are no childTopics but there are peerTopics or parentTopics? Which, in that case, should we just do `topics = [].concat(pageMetadata?.childTopics || []).concat(... || [])...`

Reply:
good catch.  ok, can I do a good'ol for loop though?  This whole `.concat(xyz || [])` feels so ugly, or is it the JS way? :)

Reply:
haha yes, you can do a for loop

Reply:
I went with your js expert suggestion after all :), the need to distinguish between null vs. empty makes it more verbose I realize now.